doc-deploy-stable/action.yml

```diff
@@ -453,7 +453,7 @@ runs:
           sed -i 's/href="\([^:"]*\)"/href="version\/dev\/\1"/g' index.html
           sed -i 's/src="\([^:"]*\)"/src="version\/dev\/\1"/g' index.html
           sed -i 's/action="search.html"/action="version\/dev\/search.html"/g' index.html
-          sed -i 's|script\.src = ".*_static/js/meilisearch_theme_wrap\.js"|script.src = "version/stable/_static/js/meilisearch_theme_wrap.js"|g' index.html
+          sed -i 's|script\.src = ".*_static/js/meilisearch_theme_wrap\.js"|script.src = "version/dev/_static/js/meilisearch_theme_wrap.js"|g' index.html
```

Comment:
Why should it be ``dev`` here instead of ``stable``? This is the doc-deploy-stable action

Reply:
We were using the same script in both the stable and development deployments, which I believe was added during the testing phase. I've since removed the script in [2d665bd](https://github.com/ansys/actions/pull/539/commits/2d665bde7c97f350feffe73a7129561d14ea3178)

Reply:
Okay makes sense now, thanks @Revathyvenugopal162